### PR TITLE
Adding warning for scala native

### PR DESCRIPTION
**Pull request by TonioGela**

### CODE REVIEW

**Comment by TonioGela** [comment]
May we wanna add a warning on the README too?

File: docs/index.md [hunk 1]
```diff
@@ -69,9 +69,39 @@ object Hello extends IOApp.Simple {
 ```
 @:@
 
-### Native Image
+### Warning
```
Comment (zetashift) [inline docs/index.md:72]:
```suggestion
### Scala Native usage
```

everything else looks good to me

**Comment by zetashift** [comment]
> May we wanna add a warning on the README too?

I feel like the intersection of toolkit users and scala native users is sorta small, and seeing the innocence of the warning, I think the current doc change is enough.

File: docs/index.md (hunk 1 above)
Comment (TonioGela) [reply docs/index.md]: Seems legit

**Review by zetashift — APPROVED** [review]

**Comment by zetashift** [comment]
Thanks! Also gave me the strong desire to try out Scala Native :P

**Pull request merged**